index.html

```diff
@@ -0,0 +1,160 @@
+          <h2 class="card-heading">Multi-Post<br>Stories</h2>
+          <ul class="tool1">
+            <li>CANOPY</li>
+            <li><img src="image/Counter.png" alt="dot"></li>
+            <li class="fint">Back End Dev</li>
+            <li><img src="image/Counter.png" alt="dot"></li>
+            <li class="fint">2015</li>
+          </ul>        
+          <div class="detail-style1">A daily selection of privately<br>reads:no account or<br>sign-up required</div>
+          <ul class="tool2">
+            <li><a href="#">html</a></li>
+            <li><a href="#">css</a></li>
+            <li><a href="#">javascript</a></li>
+          </ul>
+            <button type="button" class="button detail-style2">See Project</button>
+        </div>
+      </div>
+      </section>
+      <section class="about-section">
+        <div class="myself">
+          <h2>About<br>Myself</h2>
+          <p class="about-detail">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+          <p><a href="" class="info-link">LET'S CONNECT</a></p>
+          <ul class="about-social">
+            <li><a href="#"><img src="image/Vector-(4).png" alt="Email icon"></a></li>
+            <li><a href="#"><img src="image/Linkedin-icon.png" alt="Social media"></a></li>
+            <li><a href="#"><img src="image/Vector-(1).png" alt="Email icon"></a></li>
+            <li><a href="#"><img src="image/Vector-(2).png" alt="Social media"></a></li>
+            <li><a href="#"><img src="image/Vector-(3).png" alt="Social media"></a></li>
+          </ul>
+          <button type="button" class="button my-resume">Get my resume</button>
+        </div>
+        <div class="language-detail">
+          <ul class="lang icon">
+            <li class="word">Language</li>
+            <li class="size"><img src="image/arrow2.png" alt="dropdown arrow icon"></li>          
+          </ul>
+          <ul class="icons">
+            <li><img src="image/Ellipse-1.png" alt="Javascript logo" class="spac"></li>
+            <li class="spac"><a href="#">Javascript</a></li>
+          </ul>
+          <ul class="icons">
+            <li class="spac"><img src="image/Ellipse-1-(1).png" alt="HTML logo"></li>
+            <li class="spac"><a href="#">HTML</a></li>
+          </ul>
+          <ul class="icons">
+            <li class="spac"><img src="image/Ellipse-1-(2).png" alt="CSS logo"></li>
+            <li class="spac"><a href="#">CSS</a></li>
+          </ul>
+          <ul class="lang icon">
+            <li class="word">Frameworks</li>
+            <li class="word"><img src="image/arrow2.png" alt="forward arrow icon"></li>
+          </ul>
+          <ul class="lang icon">
+            <li class="word">Skills</li>
+            <li><img src="image/arrow2.png" alt="forward arrow icon"></li>
+          </ul>
+        </div>
```

Comment:
- [ ] Great job implementing the language categories, but it's better to put them inside a `ul` and each category in a `li` to me more sematic. Please change the `div` to a `ul` tag.